packages/cosmic-swingset/src/block-manager.js

```diff
@@ -162,34 +173,32 @@ export default function makeBlockManager({
         verboseBlocks && console.info('block', action.blockHeight, 'begin');
 
         // Start a new block, or possibly replay the prior one.
-        const leftoverActions = currentActions.filter(
-          a => a.blockHeight !== action.blockHeight,
-        );
-        if (leftoverActions.length) {
+        const leftovers = new Map();
```

Comment:
Maybe rename to `leftoverTypes` to make it clear that we're aware of the possibility of e.g. multiple `DELIVER_INBOUND` in a single block. This code doesn't need to see all the actions, it's only gathering the type names to build a debugging message.